drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -286,10 +286,6 @@ msgid ""
 msgstr ""
 
 #: templates/html/js_templates.html
-msgid "Zones"
```

Comment:
@e-kolpakov Thanks for cleaning up the translations! How did you do that? Did you use gettext CLI tools to extract the strings? It would be great if you could document the commands you used to do that somewhere.